Utility/PiecesTemplates.FCMacro

```diff
@@ -0,0 +1,421 @@
+        "Nombre de la Pieza:",
+        "Información de Uso",
+        "Grupos/Directorios a crear:",
+        "Crear Estructura de Pieza",
+        "Versión:",
+        "Descripción:",
+        "Información",
+        "Este script (macro) genera una estructura de grupos (directorios) en una pieza, de manera que sea más fácil organizar las distintas operaciones,referencias y objetos de esa pieza.",
+        "Los grupos inicialmente están predefinidos, pero pueden ser cambiados fácilmente.",
+        "Operaciones sobre Grupos",
+        "Añadir Grupo",
+        "Eliminar Grupo",
+        "el primero",
+        "el último",
+        "Listado Piezas Creadas:",
+        "Nº de Piezas Creadas:"
+
+    ],
+    [ # English
+        ## Strings for translation into any language (English Strings)
+        "Part Template Creator",
+        "The document is not active",
+        "Creation of Parts",
+        "Part Name:",
+        "Usage Information:",
+        "Groups/Directories to create:",
+        "Create Part Structure",
+        "Version:",
+        "Description:",
+        "Information",
+        "This script (macro) generates a structure of groups (directories) in a part, so that it is easier to organize the different operations, references, and objects of that part.",
+        "The groups are initially predefined, but can be easily changed.",
+        "Operations on Groups",
+        "Add Group",
+        "Delete Group",
+        "first",
+        "the last",
+        "List Pieces Created:",
+        "Number of Pieces Created:"
+
+    ],
+    [ # Italiano
+        ## Strings for translation into any language (Italian strings)
+        "Creatore di strutture delle Parti",
+        "Il documento non è attivo",
+        "Creazione delle Parti",
+        "Nome della Parte:",
+        "Informazioni per l'uso:",
+        "Gruppi/Cartelle da creare:",
+        "Crea la struttura della Parte",
+        "Versione:",
+        "Descrizione:",
+        "Informazioni",
+        "Questo script (macro) genera una struttura di gruppi (cartelle) in una nuova parte, in modo che sia più semplice organizzare le varie operazioni, i riferimenti e gli elementi di questa parte.",
+        "I gruppi inizialmente risultano predefiniti, però si possono modificare facilmente.",
+        "Operazioni sui Gruppi",
+        "Aggiungi Gruppo",
+        "Elimina Gruppo",
+        "per primo",
+        "per ultimo",
+        "Lista delle parti create:",
+        "Nº di parti create:" 
+    ]
+        ]
+
+ui_path = app.getUserMacroDir(True) + "/PiecesTemplates/ui/PiecesTemplates.ui"
+gr_path = app.getUserMacroDir(True) + "/PiecesTemplates/Groups.txt"
+
+try:
+    _fromUtf8 = QtCore.QString.fromUtf8
+except AttributeError:
+    def _fromUtf8(s):
+        return s
+
+class PiecesTemplates:
+
+    def __init__(self):
+        self.doc = app.activeDocument()
+        if not self.doc:
+            app.Console.PrintWarning(tr('El documento no está activo') + cIntro)
```

Comment:
I don't really get the purpose of `cIntro` why not write `\n` directly? Moreover, it looks more like  a conclusion than an introduction.